databricks/koalas/series.py

```diff
@@ -1115,6 +1115,71 @@ def rename(self, index=None, **kwargs):
         else:
             return first_series(kdf)
 
+    def rename_axis(
+        self, mapper: Optional[Any] = None, index: Optional[Any] = None, inplace: bool = False
+    ) -> Optional["Series"]:
+        """
+        Set the name of the axis for the index or columns.
+
+        Parameters
+        ----------
+        mapper, index :  scalar, list-like, dict-like or function, optional
+            A scalar, list-like, dict-like or functions transformations to
+            apply to the index values.
```

Comment:
Maybe this it's also not correct? I think you can refer to [here](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.rename_axis.html#pandas-series-rename-axis).

Reply:
This is for the same reason I explained above.